main.js

```diff
@@ -0,0 +1,244 @@
+var moment = require('moment'); // require
```

Comment:
This folder structure is fine as it is.


However, let's make it more organized, because this project will grow.

Usually all the code lives under a folder `src/` (the styles, the js, the HTML, etc...)
So, move those to that folder.

Then you need to tell parcel to run in the src folder instead of root directory. You dod that by adding a script to `package.json`:

```diff
  "scripts": {
+    "dev": "parcel src/index.html",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
```

Finally you can run `npm run dev` to run the project in dev mode